ms/common_setup.cmd

```diff
@@ -1,13 +1,19 @@
 @echo off
+setlocal EnableDelayedExpansion
 
+set CUSTOM_BUIL_DIR=%~1
+if "%CUSTOM_BUIL_DIR%"=="" (
 set BUILD_DIR="%SDL_ROOT%\build"
```

Comment:
see 9ebec2c

________________________________

This e-mail and any attachment(s) are intended only for the recipient(s) named above and others who have been specifically authorized to receive them. They may contain confidential information. If you are not the intended recipient, please do not read this email or its attachment(s). Furthermore, you are hereby notified that any dissemination, distribution or copying of this e-mail and any attachment(s) is strictly prohibited. If you have received this e-mail in error, please immediately notify the sender by replying to this e-mail and then delete this e-mail and any attachment(s) or copies thereof from your system. Thank you.